kalliope/signals/geolocation/model.py

```diff
@@ -0,0 +1,17 @@
+class Geolocation(object):
```

Comment:
Missing serialize class? Like [here](https://github.com/kalliope-project/kalliope/blob/dev/kalliope/signals/mqtt_subscriber/models.py#L13).
Not needed?

Reply:
No need to overwrite the default one in this case.